tests/pages/storage-page.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class StoragePage {
+    readonly page: Page;
+    readonly encryptionSlider: Locator;
+    readonly encryptionEnabled: Locator;
+    readonly backButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.encryptionSlider = page.locator('#encryption-off');
+        this.backButton = page.getByRole('button', { name: 'Back' });
+        this.encryptionEnabled = page.getByRole('button', { name: 'Encryption settings' });
+    }
+
+    async useEncryption() {
+        await this.encryptionSlider.click();
```

Comment:
this code is not easy to understand unless with have here something like `this.useEncryptionOffToggle` so the user can know the expectation of a toggle off where we click, without that, the toggle could be thought to be on, we cannot refer to the slider directly like this, we would need another more clear logic for that.